packages/core/examples/stackPosition/index.ts

```diff
@@ -0,0 +1,269 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  getRenderingEngine,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addButtonToToolbar,
+  addDropdownToToolbar,
+  camera as cameraHelpers,
+  ctVoiRange,
+  getLocalUrl,
+} from '../../../../utils/demo/helpers';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const { ViewportType, Events } = Enums;
+
+// ======== Constants ======= //
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'CT_STACK';
```

Comment:
trying the example, clicking on the top left change the VOI https://deploy-preview-930--cornerstone-3d-docs.netlify.app/live-examples/stackposition